From 9d51027c228dafd7db2d0cffca1f0fc695e950fd Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Mon, 2 Dec 2013 11:33:26 -0200
Subject: [PATCH] geos-config: Add includedir variable

This fixes cross-compile as it is easier to mangle the includedir
during sysroot generation.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 tools/geos-config.in |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/geos-config.in b/tools/geos-config.in
index 9b45b5f..1749892 100644
--- a/tools/geos-config.in
+++ b/tools/geos-config.in
@@ -1,6 +1,7 @@
 #!/bin/sh
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+includedir=@includedir@
 libdir=@libdir@
 
 usage()
@@ -38,7 +39,7 @@ case $1 in
     echo @VERSION@
      ;;
     --cflags)
-    echo -I${prefix}/include 
+    echo -I${includedir}
       ;;
     --libs)
       # TODO: make an alias for --clibs
-- 
1.7.10.4

